### ISSUE TITLE
Javascript Console/Workspace in Antetype

### ISSUE THREAD

**Issue by esdominikb** [esdominikb/antetype-goes-js#5]
It would be greate if we have a Workspace or Console (single and/or multiline) for Javascript in Antetype's 'Develop' menu.

@fizfaz do you think this is easy to make?

**Comment by fizfaz** [comment]
yes, should be easy to do, but before I would like to use CocoaScript instead of JSTalk …
